internal/models/schema-recipe.go

```diff
@@ -911,26 +897,36 @@ func (n *NutritionSchema) UnmarshalJSON(data []byte) error {
 				}
 			}
 		}
+	default:
+		slog.Warn("Could not parse nutrition schema", "schema", v, "type", x)
+	}
+	return nil
+}
 
-		if val, ok := x["sodiumContent"].(string); ok {
-			n.Sodium = val
-		}
-
-		if val, ok := x["sugarContent"].(string); ok {
-			n.Sugar = val
-		}
+// EnsureNutritionUnitForString extracts digits from nutritional property and appends metric unit.
+// Returns a string value.
+func EnsureNutritionUnitForString(nutritionValue any, nutritionProperty string) string {
+	var nutritionPropertyLowerCase = strings.ToLower(nutritionProperty)
+	var nutritionString string = extensions.ConvertToString(nutritionValue)
+	var nutritionStringDigits string = regex.Digit.FindString(nutritionString)
```

Comment:
We can use the short way to declare the variables by using the assignment operator, i.e.
```go
	nutritionPropertyLowerCase := strings.ToLower(nutritionProperty)
	nutritionString := extensions.ConvertToString(nutritionValue)
	nutritionStringDigits := regex.Digit.FindString(nutritionString)
```

Reply:
Changed.

What is best/common practice in such cases?
I thought it was prudent to type the variables directly instead of being inferred from the called functions, as we expect these variables to always be strings down the line.
In this case they will of course still be strings anyway, so I can see the case for both variants.